README.md

```diff
@@ -44,6 +44,15 @@ php8.1-debug-v1
 ...
 ```
 
+#### Configure Xdebug in your IDE
+
+![How to configure Xdebug with Pimcore Docker image](./doc/xdebug.png)
+
+#### Use step-debugging
+
+- **Browser**: Install [Xdebug helper](https://chrome.google.com/webstore/detail/xdebug-helper/eadndfjplgieldjbigjakmdgkmoaaaoc) browser extension and start debugging session in your tab. 
+- **CLI**: Run your command like this: `docker compose exec -e XDEBUG_TRIGGER=PHPSTORM php bin/console`
```

Comment:
I use that as env var in the docker-compose debug file and don't need the trigger. Also, doesn't need to be `docker compose exec *php-debug* -e ...`?

```
    environment:
      - PHP_IDE_CONFIG=serverName=localhost
```

Reply:
Well, depends, if you don't want to run every command with Xdebug (performance), using `docker-compose.yml` is not so good. 

Regarding `php-debug`: No, we recently changed the default `docker-compose.yml` because we have only one container for PHP, see: https://github.com/pimcore/demo/blob/11.x/docker-compose.yaml#L28

But you're right, skeleton is missing, will do a PR 😊

Reply:
There is no `php-debug` container in the skeleton anymore.

Reply:
I'm sorry, skeleton is not missing, it was just not merged to main branch yet - done now 😊

Reply:
But yeah, fine for me, we don't use the Pimcore skeleton anyway ;)

Reply:
Xdebug 3 shouldn't do much stuff when not triggered: https://php.watch/articles/xdebug2-vs-3-benchmark#footprint

Reply:
@dpfaffenbauer yep, see also: https://github.com/pimcore/skeleton/pull/99#issue-1284058179
Performance is fine now with Xdebug 3 when not triggered, at least on Win/Linux, where it was super slow as well back with Xdebug 2.